doc/ref_cert/RC2/index.rst

```diff
@@ -0,0 +1,55 @@
+Reference Conformance - Kubernetes based platform for Telecom
+=============================================================
+
+This is the Kubernetes Based Reference Conformance (RC-2)
+
+Release Information
+-------------------
+
+**Bundle: 6**
```

Comment:
does this info need to be updated

Reply:
Yes well catch- thx . New changes as I do modify md first.